bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/util/exceptions/BallerinaErrorReasons.java

```diff
@@ -20,12 +20,7 @@
 import io.ballerina.runtime.api.utils.StringUtils;
```

Comment:
Do we still need any usages of this class?

Reply:
Yes for now, there are few errors needed to be migrated to new error creator api.